index.bs

```diff
@@ -72,8 +72,13 @@ spec: web-otp; urlPrefix: https://wicg.github.io/web-otp
   type: interface
     text: OTPCredential; url: otpcredential
 spec: FEDCM; urlPrefix: https://fedidcg.github.io/FedCM
+  type: dfn
+    text: identity-credentials-get
   type: interface
     text: IdentityCredential; url: identitycredential
+  type: dict-member
```

Comment:
I guess we should export these from FedCM so they can automagically be used here... but this is fine for now.

Reply:
Yeah, I'm curious why they don't get picked up, I tried exporting the PP name. The spec works for now, but I'll ask someone wiser than me what we're doing wrong here..

Reply:
Talked to wise sage @tabatkins who will add FedCM to the spec database for us(in a grand ceremony, I expect). That should fix this.

Thank you, Tab!

Reply:
Added now, give the database an hour or so for the next run to pick up the definitions.